cheatsheets_draft/Legacy_Application_Management_Cheat_Sheet.md

```diff
@@ -0,0 +1,75 @@
+# Legacy Application Management Cheat Sheet
+
+## Introduction
+
+Legacy applications are applications that are recognised as being outdated but remain in active use by an organisation. This may occur if a viable alternative to the application is not available, replacement is currently too expensive, or if the application is highly bespoke and services a very niche role within an organisation's digital ecosystem. Legacy applications often introduce significant security risks to an organisation for the following reasons:
```

Comment:
```suggestion
Legacy applications are applications that are recognized as being outdated but remain in active use by an organization. This may occur if a viable alternative to the application is not available, replacement is currently too expensive, or if the application is highly bespoke and services a very niche role within an organization's digital ecosystem. Legacy applications often introduce significant security risks to an organization for the following reasons:
```
I'm in favor of standardizing on US English spellings. It avoids confusion and makes searching easier.

Reply:
Sounds very reasonable. I was 70% sure that might be a suggestion but wasn't sure how consistent US vs British spelling was across the others. Adjusted.